### PR TITLE
jira report log change

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

File: src/main/proto/Report.proto [hunk 1]
```diff
@@ -127,4 +127,7 @@ message JiraTickets {
     string priority = 6;
     string reporter = 7;
     string summary = 8;
+    string viewTicket = 9;
+    string creator = 10;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: What is the use of creator? Can we rename it to createdBy

File: src/main/proto/ReportService.proto [hunk 2]
```diff
@@ -164,6 +164,12 @@ message getAllJiraRequest{
 message getAllJiraResponse{
     repeated JiraTickets records = 1;
 }
+message ignoreTicketCreationRequest{
+    int32 id =1;
+}
```
Comment (johibkhan2) [inline src/main/proto/ReportService.proto:169]: Can you rename it to disableTicketCreation?

File: src/main/proto/ReportService.proto [hunk 3]
```diff
@@ -164,6 +164,12 @@ message getAllJiraRequest{
 message getAllJiraResponse{
     repeated JiraTickets records = 1;
 }
+message ignoreTicketCreationRequest{
```
Comment (ir73) [inline src/main/proto/ReportService.proto]: any reason some messages are upper case and some lower case?

File: src/main/proto/ReportService.proto [hunk 4]
```diff
@@ -164,6 +164,12 @@ message getAllJiraRequest{
...
+    int32 id =1;
```
Comment (ir73) [inline src/main/proto/ReportService.proto]: space after `=` is missing

File: src/main/proto/ReportService.proto (hunk 3 above)
Comment (ir73) [inline src/main/proto/ReportService.proto]: I see other messages have a JIRA word in their name, but this one does not. Not sure if it's good for consistency.

Also not sure it's the best idea to have JIRA specific messages.  What if we add trello integration, are we going to have another 10 trello specific messages?

File: src/main/proto/ReportService.proto (hunk 4 above)
Comment (ir73) [inline src/main/proto/ReportService.proto]: not clear what ID is that, could it be `ticketId`?

**Review by ir73 — CHANGES_REQUESTED** [review]

File: src/main/proto/Report.proto [hunk 5]
```diff
@@ -127,4 +127,7 @@ message JiraTickets {
     string priority = 6;
     string reporter = 7;
     string summary = 8;
+    string viewTicket = 9;
```
Comment (ir73) [inline src/main/proto/Report.proto:130]: should be snake case based on the convention https://developers.google.com/protocol-buffers/docs/style#message-and-field-names

File: src/main/proto/ReportService.proto [hunk 6]
```diff
@@ -149,20 +149,29 @@ message JiraConfigsRequest {
...
 
-message getAllJiraRequest{
+message GetAllJiraRequest {
     string status = 1;
+    int32 pageSize = 2;
+    int32 page = 3; 
+}
+message GetAllJiraResponse {
+    int32 totalRecords = 1;
+    repeated JiraTickets records = 2;
```
Comment (ir73) [inline src/main/proto/ReportService.proto:168]: shouldn't `JiraTickets` be in singular as `JiraTicket`? I believe the message `JiraTickets` represents only one ticket, isn't it?

Comment (khushbu-maurya) [reply src/main/proto/ReportService.proto:168]: no, its list of created jira

File: src/main/proto/ReportService.proto [hunk 7]
```diff
@@ -149,20 +149,29 @@ message JiraConfigsRequest {
...
+}
+message DisableTicketCreationRequest { 
+    int32 schedulerTaskLogs = 1;
```
Comment (johibkhan2) [inline src/main/proto/ReportService.proto]: rename it to schedulerTaskLogId

File: src/main/proto/ReportService.proto [hunk 8]
```diff
@@ -149,20 +149,29 @@ message JiraConfigsRequest {
...
 }
-message getAllJiraResponse{
-    repeated JiraTickets records = 1;
+message DisableTicketCreationResponse {
```
Comment (johibkhan2) [inline src/main/proto/ReportService.proto]: U can reuse ConfigResposne message

**Review by johibkhan2 — APPROVED** [review]

**Pull request merged**